cast/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! Cast
 //!
 //! TODO
+use async_stream::stream;
```

Comment:
don't think we need this?

Reply:
Lots of things we no longer need. Currently trying to nail down the solution before I refine it for prime time release. Pushed more so that I can get an "OK" on the direction that I am taking this.